pkg/ccl/backupccl/testdata/backup-restore/in-progress-imports

```diff
@@ -314,273 +314,3 @@ SELECT * FROM d.foofoo;
 ----
 1 x
 10 x0
-
-
-#######################
-# Version Gate Testing
-#######################
-
-# In an unfinalized cluster, back up some in-progress imports, and ensure that once the tables come
-# back online, we fully back them again, even if the cluster is not fully upgraded. Test on cluster
-# and database backups.
-#
-# Note that during IMPORT planning on an unfinalized cluster, the
-# ImportStartTime is not bound to the table's descriptor, therefore during
-# RESTORE AOST in-progress IMPORT, these tables should get thrown out.
-#
-# TODO(msbutler): cover mixed version RESTORE TABLE
-#
-# Disabled to run within tenants because version gating infra does not play nice within tenants.
-# More investigation required.
-
-
-new-cluster name=s4 share-io-dir=s1 allow-implicit-access beforeVersion=Start22_2 disable-tenant
```

Comment:
The test relied on a cluster not having run 22.2 migrations. This is no longer relevant as our min binary version is 22.2 which means all 22.2 migrations have indeed run.

Reply:
sad to see this noble subtest go :'(

but it's for the better.

Reply:
It was a 💯 test, it will live on in release-22.2 for eternity